packages/viewer/src/rendering/remote.ts

```diff
@@ -51,10 +67,27 @@ function requestFrame(api: StreamApi): RemoteRenderer {
   return req => {
     const corrId = req.correlationId || UUID.create();
     const timeout = req.timeoutInMs || DEFAULT_TIMEOUT_IN_MS;
+    const animation =
+      req.animation && req.animation.milliseconds
+        ? {
+            duration: {
+              nanos: (req.animation.milliseconds % 1000) * 1000000,
+              seconds: req.animation.milliseconds / 1000,
+            },
+            easing: req.animation?.easing
+              ? easingMap[req.animation.easing]
+              : undefined,
```

Comment:
What happens if easing is undefined?

Reply:
I used the same behavior in ISS and it defaults to a linear interpolation.

Reply:
Addressed by making easing a required property on the animation.